Updated currency converter

RaikoZA opened the pull request:
Updated to ES6, added package.json with dependencies, added gitignore
file

RaikoZA committed:
Updated converter

Updated to ES6, added package.json with dependencies, added gitignore
file

RaikoZA committed:
Shortened roundOff

Instead of having variables with returning a result, ES6 returns a
result already

ShaunEgan commented on .gitignore:
```diff
@@ -0,0 +1,8 @@
+node_modules
+
+nbproject/
```
These are personal tooling ignores, they should be in your global `.gitignore`

ShaunEgan commented on currencyConverter.js:
```diff
@@ -1,39 +1,39 @@
-var request = require('request');
-var cheerio = require('cheerio');
+"use strict";
```
use single quotes

ShaunEgan commented on currencyConverter.js:
```diff
@@ -1,39 +1,39 @@
   request({ url: currencyUrl }, function (err, response, body) {
     if (err) {
-      callback(err);
+      callback('There was an error retrieving url:', err);
```
you can remove the `, err` part

ShaunEgan commented on currencyConverter.js:
```diff
@@ -1,39 +1,39 @@
+    let url = `http://themoneyconverter.com/${fromCurrency}/${toCurrency}.aspx`;
+
+    return sendRequest(url, callback);
+};
```
add newline at end of file

ShaunEgan commented on package.json:
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "currency-converter",
+  "description": "This is a currency to currency module created in NodeJS. It's purpose is to take one currency format and convert it to another currency and return the result.",
```
This could just be `"description": "Converts between currencies"`

ShaunEgan commented on package.json:
```diff
@@ -0,0 +1,27 @@
+  "version": "1.0.1",
+  "main": "currencyConverter.js",
+  "dependencies": {
+    "request": "*",
```
specify a version for these two dependencies. You can get the latest version number from `npmjs.com`

ShaunEgan reviewed (changes requested):
Good job!

RaikoZA committed:
Removed personal ignores

RaikoZA committed:
Single quotes

Added single quotes around `use strict`

RaikoZA committed:
Removed error argument

There is no need for the error to passed through to callback as its
being handled already by a custom error message

RaikoZA committed:
Added newline/Removed return

RaikoZA committed:
Shorter description

Added shorter description

RaikoZA committed:
Added versions to dependencies

RaikoZA committed:
Closing bracket

Missed a closing bracket

ShaunEgan commented on currencyConverter.js:
```diff
@@ -1,39 +1,39 @@
+const sendRequest = (currencyUrl, callback) => {
+  request({ url: currencyUrl }, function (error, response, body) {
+    if (error) {
+      callback('There was an error retrieving url';
```
missing `)`

ShaunEgan reviewed (approved):
Nice work!
